### PR TITLE
Pathfinder System - Implement Face

### PR DESCRIPTION
**Pull request by LegacyKing**

### CODE REVIEW

File: data/pathfinder/alluria_publishing/remarkable_races/rr_races.lst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Zif											STARTFEATS:1	SIZE:M	MOVE:Walk,20																LANGBONUS:TYPE=Spo
...
-Rift Swarm										STARTFEATS:1			MOVE:Walk,20				FACE:10	REACH:0																																																	BONUS:VAR|HasLowlightVision|1																																						ABILITY:Internal|AUTOMATIC|Race Traits ~ Rift Swarm												NATURALATTACKS:Swarm,Weapon.Melee,*1,4d6,SPROP=plus Poison (Fort DC 26)															RACETYPE:Outsider			RACESUBTYPE:Swarm						TYPE:Outsider		CR:14			SOURCEPAGE:p.140
+Rift Swarm										STARTFEATS:1	SIZE:L	MOVE:Walk,20						REACH:0																																																	BONUS:VAR|HasLowlightVision|1																																						ABILITY:Internal|AUTOMATIC|Race Traits ~ Rift Swarm												NATURALATTACKS:Swarm,Weapon.Melee,*1,4d6,SPROP=plus Poison (Fort DC 26)															RACETYPE:Outsider			RACESUBTYPE:Swarm						TYPE:Outsider		CR:14			SOURCEPAGE:p.140
```
[the remark this answers is not in the document]
Comment (LegacyKing) [reply data/pathfinder/alluria_publishing/remarkable_races/rr_races.lst]: SIZE:L does. But it also implies a reach of 10, whereas Swarms have a REACH of 0

File: data/pathfinder/dreamscarred_press/psionics_expanded/psionics_expanded_races.lst [hunk 2]
```diff
@@ -6,8 +6,8 @@ SOURCELONG:Psionics Expanded	SOURCESHORT:PsiEx	SOURCEWEB:http://dreamscarredpres
...
-Astral Warrior	STARTFEATS:0	SIZE:M	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision (60')	BONUS:COMBAT|AC|2|TYPE=NaturalArmor	BONUS:STAT|STR|6	BONUS:STAT|DEX|2	BONUS:STAT|CON|2																																	LEGS:2	HANDS:2												MONSTERCLASS:Astral Warrior:1			RACETYPE:Construct							SOURCEPAGE:p.66	KIT:1|Astral Warrior
+Astral Swarm				SIZE:D	MOVE:Walk,30	MODIFY:FACE|SET|10,0|PRIORITY=200	REACH:0					BONUS:COMBAT|AC|2|TYPE=NaturalArmor	BONUS:STAT|STR|-9	BONUS:STAT|DEX|8				BONUS:VAR|Mindless|1	BONUS:WEAPONPROF=Swarm|DAMAGE|-1*(STR)	ABILITY:Special Ability|AUTOMATIC|Distraction ~ Swarm|Ectoplasmic Poison ~ Ectoplasmic Swarm|No Combat Maneuvers|Swarm Traits					NATURALATTACKS:Swarm,Weapon.Natural.Melee.Swarm.Poison,*1,4d6	MONSTERCLASS:Construct:14			RACETYPE:Construct	TEMPLATE:Swarm	CR:7			SOURCEPAGE:p.66
```
[the remark this answers is not in the document]
Comment (LegacyKing) [reply data/pathfinder/dreamscarred_press/psionics_expanded/psionics_expanded_races.lst]: I believe this keeps the output to what it exists as it appears today (means I'm not fixing unit tests).

File: data/pathfinder/alluria_publishing/remarkable_races/rr_races.lst (hunk 1 above)
[the remark this answers is not in the document]
Comment (LegacyKing) [reply data/pathfinder/alluria_publishing/remarkable_races/rr_races.lst]: D'oh, I see what you mean... had the reverse thought.

**Comment by evilpixie87** [comment]
Did this also get applied to the new sources you're moving over?

**Comment by LegacyKing** [comment]
No, I haven't touched those.

**Pull request closed without merging**